### PR TITLE
[Main] Migrate Crypto to Cryptography for clicknload

### PR DESCRIPTION
**Pull request by OzzieIsaacs**
Fixed missing parameter thread

### Describe the changes

During debugging the addon_thread is complaing about unexpected keyword argument 'thread'
1) By adding the parameter thread to forward, _server and proxy function in clicknload this can be avoided

2) The clicknload decrypting routine weren't working with Fernet, I could not make it run, (Is fernet and AES the same?). I migrated it to cryptography following your example from dlc.py. I send @GammaC0de an example link via e-mail

### Is this related to a problem?

```
[2020-10-25 10:40:10]  DEBUG         pyload.webui  Fernet key must be 32 url-safe base64-encoded bytes.
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.8/dist-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ozzie/.local/lib/python3.8/site-packages/pyload/webui/app/blueprints/cnl_blueprint.py", line 30, in wrapper
    return func(*args, **kwargs)
  File "/home/ozzie/.local/lib/python3.8/site-packages/pyload/webui/app/blueprints/cnl_blueprint.py", line 108, in addcrypted2
    obj = Fernet(key)
  File "/home/ozzie/.local/lib/python3.8/site-packages/cryptography/fernet.py", line 36, in __init__
    raise ValueError(
ValueError: Fernet key must be 32 url-safe base64-encoded bytes.

```

### CODE REVIEW

File: src/pyload/webui/app/blueprints/cnl_blueprint.py [hunk 1]
```diff
@@ -48,7 +51,7 @@ def add():
         "package", flask.request.form.get("source", flask.request.form.get("referer"))
     )
 
-    urls = [url for url in flask.request.form["urls"].split("\n") if url.strip()]
+    urls = [url for url in flask.request.form["urls"].replace(' ', '\n').split("\n") if url.strip()]
```
Comment (GammaC0de) [inline src/pyload/webui/app/blueprints/cnl_blueprint.py:54]: why this change?

Comment (OzzieIsaacs) [reply src/pyload/webui/app/blueprints/cnl_blueprint.py:54]: @GammaC0de : I wrote you an email regarding this topic.

Comment (GammaC0de) [reply src/pyload/webui/app/blueprints/cnl_blueprint.py:54]: Does JDownloader handle this correctly?

**Comment by GammaC0de** [comment]
About complaining about unexpected keyword argument 'thread':

This should not happen unless you are using non English python, see [addon_thread.py#L52](https://github.com/pyload/pyload/blob/develop/src/pyload/core/threads/addon_thread.py#L52).

File: src/pyload/webui/app/blueprints/cnl_blueprint.py (hunk 1 above)
Comment (OzzieIsaacs) [reply src/pyload/webui/app/blueprints/cnl_blueprint.py:54]: Yes it does:
grafik

grafik

**Comment by OzzieIsaacs** [comment]
Regarding the 'thread' keyword, it only occurred at stepping through the code in debug mode. In "normal operation" there was no problem
